src/components/RegionInput.js

```diff
@@ -19,52 +20,68 @@ export const RegionInput = ({
   // Generate autocomplete options for regions from regionInfo
   // Add : to pathNames
   console.log("rendering with pathnames: ", pathNames);
-  const pathNamesColon = pathNames.map((name) => name + ":");
+  const pathNamesColon = pathNames.map((name) => {
+    return {label: name + ":", value: name + ":"};
+  });
   const pathsWithRegion = [];
 
+  const regionToDesc = new Map();
+
   if (regionInfo && !isEmpty(regionInfo)) {
     // Stitch path name + region start and end
     for (const [index, path] of regionInfo["chr"].entries()) {
-      pathsWithRegion.push(
-        path + ":" + regionInfo.start[index] + "-" + regionInfo.end[index]
-      );
+      const pathWithRegion = path + ":" + regionInfo.start[index] + "-" + regionInfo.end[index]
+      pathsWithRegion.push({
+        label: pathWithRegion + " " + regionInfo.desc[index],
+        value: pathWithRegion
+      });
+      regionToDesc.set(pathWithRegion, regionInfo.desc[index]);
     }
-
-    // Add descriptions
-    pathsWithRegion.push(...regionInfo["desc"]);
-
   }
 
   // Autocomplete selectable options
-  const displayRegions = [...pathsWithRegion, ...pathNamesColon];
+  const displayRegions = [...pathsWithRegion, ...pathNamesColon].filter(option => option.label !== "none:");
```

Comment:
I don't think this is the Right Way to get rid of this extra option, because the code does not convince me that we actually know why it is showing up.

Either we should get rid of it where it is actually being generated, or we should have a comment here explaining why it is there and why this actually is the best time to remove it.